installing/installing_oci/installing-oci-assisted-installer.adoc

```diff
@@ -51,7 +51,9 @@ include::modules/installing-troubleshooting-assisted-installer-oci.adoc[leveloff
 [role="_additional-resources"]
 .Additional resources
 
+* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/openshift-troubleshooting.htm[Troubleshooting {product-title} on {oci} (Oracle documentation)]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
Did you intend to have two `.Additional resources` headings on top of each other?

Reply:
Well spotted. Just one. Removed the duplicate.